### ISSUE TITLE
cluster-ui: custom metrics page fails to load completely in 21.1.2

### ISSUE THREAD

**Issue by dhartunian** [cockroachdb/cockroach#66781]
**Describe the problem**

Attempting to load `/#/debug/chart` results in a blank page:

**To Reproduce**

roachprod commands:

```
roachprod create local -n 4
roachprod stage local release v21.1.2
roachprod start local
roachprod adminui local
```

Then navigate to advanced debug and click custom charts

**Expected behavior**
Expect to see custom charts page

**Additional data / screenshots**
![Screen Shot 2021-06-23 at 1 15 27 PM](https://user-images.githubusercontent.com/986307/123140402-3ef42c00-d425-11eb-902d-581bc4240e5b.png)

**Environment:**
 - Affected CockroachDB versions: 21.1.2, 21.1.3, 21.1.4

gz#8808

**Comment by smcvey** [comment]
Also affects 21.1.3 and 21.1.4.

**Comment by thtruo** [comment]
https://github.com/cockroachdb/cockroach/issues/66890 resolves this issue but the fix missed the release train for 21.1.4 unfortunately; it should be part of the 21.1.5 point release

**Comment by jordanlewis** [comment]
@thtruo we should close this issue if it is resolved.
